src/nft/components/explore/Carousel.tsx

```diff
@@ -37,36 +32,32 @@ const CarouselCardContainer = styled.div`
 
 const CarouselItemCard = styled(a.div)`
   ${carouselHeightStyle}
-  ${carouselItemStyle}
 
   display: flex;
   justify-content: center;
-  padding: 3px 32px 32px 32px;
-
+  padding: 4px 12px 32px;
   position: absolute;
   will-change: transform;
+
+  @media screen and (min-width: ${({ theme }) => theme.breakpoint.lg}px) {
+    padding: 4px 32px 32px;
+  }
 `
 
 const CarouselItemIcon = styled.div`
   ${carouselHeightStyle}
-  ${carouselItemStyle}
-
-  display: flex;
   align-items: center;
-  padding-left: 8px;
-  padding-right: 8px;
+  color: ${({ theme }) => theme.textPrimary};
   cursor: pointer;
+  display: none;
+  padding-left: 4px 8px 32px;
```

Comment:
Did you mean `padding` here? Also, previously the top/bottom paddings were shared between an icon and a card, which I think is quite useful as they depend on each other.

If this is `padding`, this is pretty much equivalent to previous:
```
padding-left: 8px;
padding-right: 8px;
```
+ shared `carouselItemStyle` adding 4px/32px top/bottom paddings. 

So yeah, I think my conclusion is I'd love for these paddings to stay shared between an item and a card so it is easier to manage them in the future.

Reply:
yes, ty

Reply:
see: https://github.com/Uniswap/interface/pull/5300#discussion_r1026588984  regarding re-use in this case